cql/ManageSpecialPopulation.cql

```diff
@@ -217,10 +217,20 @@ define HistologyInterpretedAsLessThanCin2AfterAbnormalCytologyScreening:
   Collate.MostRecentBiopsyReport.date 3 years or less after Rare.MostRecentCytologyReport.date
 
 define SecondMostRecentCytologyInterpretedAsAscHOrHsil:
+  SecondMostRecentCytologyInterpretedAsAscH or
+  SecondMostRecentCytologyInterpretedAsHsil
+
+define SecondMostRecentCytologyInterpretedAsAscH:
+  AnyTrue(
+    (Rare.SecondMostRecentCytologyReport.allConclusions) aC
+      return
+        aC ~ "ASC-H"
+  )
+
+define SecondMostRecentCytologyInterpretedAsHsil:
   AnyTrue(
     (Rare.SecondMostRecentCytologyReport.allConclusions) aC
       return
-        aC ~ "ASC-H" or
         aC in "HSIL"
   )
```

Comment:
`SecondMostRecentCytologyInterpretedAsAscHOrHsil`,`SecondMostRecentCytologyInterpretedAsAscH`,`SecondMostRecentCytologyInterpretedAsHsil`:

Should we also put these 3 expressions over in ManageRareAbnormality.cql, with `CytologyInterpretedAsAscHOrHsil`, which you added there?

Reply:
I agree it makes sense to put these in ManageRareAbnormality. Will move.

Reply:
I still think these make more sense lower down in the ManageRareAbnormality.cql file, right after where you added `CytologyInterpretedAsAscHOrHsil`, since these expressions are very closely related. They don't really play a part in the Rare Cytology section where you moved them to, is there a reason they were added there? I won't let this hold up the PR, since functionally it doesn't change a thing but I leave it up to you if you would like to change it.

Reply:
These are no longer needed due to refactoring. They will be removed.